src/lambda/explorer/cloudFormationNodes.ts

```diff
@@ -48,13 +49,17 @@ export class DefaultCloudFormationNode extends AWSTreeNodeBase implements CloudF
         this.stackNodes = new Map<string, CloudFormationStackNode>()
     }
 
-    public async getChildren(): Promise<CloudFormationStackNode[]> {
-        await this.updateChildren()
+    public async getChildren(): Promise<(CloudFormationStackNode | ErrorNode)[]> {
+        await this.handleErrorProneOperation(async () => this.updateChildren(),
```

Comment:
Style nit: When splitting arguments into multiple lines, we indent each a single level. For example:

```typescript
await this.handleErrorProneOperation(
    async () => this.updateChildren(),
    localize(
        'AWS.explorerNode.cloudFormation.error',
        'Error loading CloudFormation resources'
    )
)
```